signer/src/testing/wallet.rs

```diff
@@ -46,6 +49,34 @@ pub fn generate_wallet() -> (SignerWallet, [Keypair; 3], u16) {
     (wallet, key_pairs, signatures_required)
 }
 
+/// This function creates a signing set where the aggregate key is the
+/// given controller's public key.
+pub fn create_signers_keys<R>(rng: &mut R, signer: &Recipient, num_signers: usize) -> Vec<PublicKey>
+where
+    R: rand::Rng,
+{
+    // We only take an odd number of signers so that the math works out.
+    assert_eq!(num_signers % 2, 1);
+
+    let private_key = PrivateKey::from(signer.keypair.secret_key());
+    let aggregate_key = PublicKey::from_private_key(&private_key);
+    // The private keys of half of the other signers
+    let pks: Vec<secp256k1::SecretKey> = std::iter::repeat_with(|| secp256k1::SecretKey::new(rng))
+        .take(num_signers / 2)
```

Comment:
I guess `num_signers` is meant to be _exlusive of_ the current signer?

Reply:
No, it is supposed to include the current signer. Why do you ask?

Reply:
Nah it was just because this isn't technically half of the other signers , but I guess that doesn't really matter here since after looking again you're asserting it's odd above which solves it 👍